.github/ISSUE_TEMPLATE.md

```diff
@@ -1,50 +1,18 @@
-<!--
-PLEASE HELP US PROCESS GITHUB ISSUES FASTER BY PROVIDING THE FOLLOWING INFORMATION.
--->
+# How to Summit an Issue?
 
-# Checklist
-<!-- Check one of the following options with "x" -->
+### Step 1: Are you in the right place?
 
-- [ ] Regression (a behaviour that used to work and stopped working in a new release)
-- [ ] Bug report <!-- Please search GitHub for a similar issue or PR before submitting -->
-- [ ] Feature request
-- [ ] Documentation issue or request
-- [ ] Support request => Please do not submit support request here, instead see our [Questions or doubts section](https://git.io/vN6fh)
+  * For issues or feature requests related to the code **in this repository** file a Github issue.
+  * For general technical questions, post a question on [StackOverflow](http://stackoverflow.com/) tagged appropriately.
+  * For general discussion, use the [Flyve-MDM mailing list](http://mail.ow2.org/wws/info/flyve-mdm-dev).
+  * For help troubleshooting for customers [support channel](https://support.teclib.com/).
+  
+### Step 2: Describe the problem:
+  
+#### Observed Results:
 
-## Current behaviour
-<!-- Describe how the issue manifests. -->
+ * This could be a description, error output, steps to reproduce, a feature missed, etc.
```

Comment:
`<!-- This could be a description, error output, steps to reproduce, a feature missed, etc. -->`